script/import_scripts/quandora/test/test_data.rb

```diff
@@ -96,7 +96,7 @@
         "author" : {
           "uid" : "204973f4-2dfe-494c-b1b2-3cd1cbac34f0",
           "name" : "204973f4-2dfe-494c-b1b2-3cd1cbac34f0",
-          "email" : "eexcited@mydomain.com",
+          "email" : "excited@mydomain.com",
```

Comment:
I should drop this. I mostly recognized that the naming convention was `flast@`.

Note that it'd be better for `@example.com` or similar be used instead of using a real domain.

Reply:
We typically use `.invalid` now, as it's special cased in the mail delivery code.